### DISCUSSION TITLE
希望加入触摸屏

### DISCUSSION THREAD

**Discussion by sonicepro** [utmapp/UTM#4534] [UTM Help (iOS)]
ipadpro装utm跑win11只能用屏幕控制鼠标，希望能加入触摸屏功能，用手指点击更方便娱乐

**Comment by ms300** [comment]
Touch mode is available but there is no global settings page when you install UTM by trollstore.
#4446

**Comment by sonicepro** [comment]
所以它属于trollStore的bug吗？能不能在utm中加个全局设置的个入口呢




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "utmapp/UTM"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月20日(星期四) 晚上7:26
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [utmapp/UTM] 希望加入触摸屏 (Issue #4525)





 
Touch mode is available but there is no global settings page when you install UTM by trollstore.
 #4424

**Comment by gaogy1853** [comment]
我也是通过trollStore安装的，非常希望加入触摸屏。
